common/views/components/GlobalContextProvider/GlobalContextProvider.tsx

```diff
@@ -1,25 +1,32 @@
 import { GetServerSidePropsContext } from 'next';
 import { createContext, FunctionComponent, ReactNode } from 'react';
-import OpeningTimesContext from '../OpeningTimesContext/OpeningTimesContext';
-import GlobalAlertContext from '../GlobalAlertContext/GlobalAlertContext';
-import PopupDialogContext from '../PopupDialogContext/PopupDialogContext';
+import OpeningTimesContext, {
+  OpeningTimes,
+} from '../OpeningTimesContext/OpeningTimesContext';
+import GlobalAlertContext, {
+  GlobalAlert,
+} from '../GlobalAlertContext/GlobalAlertContext';
+import PopupDialogContext, {
+  PopupDialog,
+} from '../PopupDialogContext/PopupDialogContext';
 import TogglesContext from '../TogglesContext/TogglesContext';
 import { parseCollectionVenues } from '../../../services/prismic/opening-times';
+import { Toggles } from '@weco/toggles';
 
-export type GlobalContextData = {
-  toggles: any;
-  globalAlert: any;
-  popupDialog: any;
-  openingTimes: any;
-};
+export interface GlobalContextData {
```

Comment:
Why `interface`? I thought we were avoiding this?

Reply:
Ah, at one point in this PR it needed to be inherited from (or something like that) but I seem to have removed that bit so will change it back